modules/ROOT/pages/keycloak/coding.adoc

```diff
@@ -19,6 +19,15 @@ dependencies {
     implementation "org.aerogear:android-auth:{release-number}"
 }
 ----
+. Additionally, to prevent build errors, also add the following in your app's *build.gradle*:
```

Comment:
`. To prevent build errors, add the following in your app's *build.gradle*: `
if the code is ok, is ok to merge

Reply:
@pmacko1 the code is fine. Feel free to merge and delete my branch. Thanks

Reply:
@pawelpaszki could you update the line with my suggestion? ill then merge it. thx